components/aws/sagemaker/tests/integration_tests/resources/definition/update_endpoint_pipeline.py

```diff
@@ -0,0 +1,71 @@
+import kfp
+from kfp import components
+from kfp import dsl
+
+sagemaker_model_op = components.load_component_from_file("../../model/component.yaml")
+sagemaker_deploy_op = components.load_component_from_file("../../deploy/component.yaml")
+
+
+@dsl.pipeline(
+    name="Create Hosting Endpoint in SageMaker",
+    description="SageMaker deploy component test",
+)
+def update_endpoint_pipeline(
```

Comment:
(optional) previously we added most fields to the pipeline definition for coverage, but these are not needed for feature tests. We can remove the parameters which are not required by the pipeline example initial_instance_count/weight, endpoint_url etc just to reduce the amount of code

Reply:
We can do this as we address this in create_endpoint. I will leave it this way for now